browser/resources/settings/brave_leo_assistant_page/memory_section.html

```diff
@@ -0,0 +1,302 @@
+  }
+
+  .memory:last-of-type {
+    border-bottom: none;
+  }
+
+  .list-container {
+    padding: var(--leo-spacing-l) var(--leo-spacing-2xl)
+  }
+
+  .action-box {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: var(--leo-spacing-m);
+    padding: var(--leo-spacing-xl) var(--leo-spacing-2xl) var(--leo-spacing-m) var(--leo-spacing-2xl);
+  }
+
+  .label-secondary {
+    font: var(--leo-font-default-regular);
+    color: var(--leo-color-text-secondary);
+  }
+
+  .header {
+    padding: var(--leo-spacing-2xl) var(--leo-spacing-2xl) 0 var(--leo-spacing-2xl);
+  }
+
+  .main-header {
+    font: var(--leo-font-heading-h4);
+  }
+
+  .memory-actions {
+    display: flex;
+    height: 36px;
+    justify-content: flex-end;
+    align-items: center;
+    gap: var(--leo-spacing-m);
+  }
+
+  settings-toggle-button {
+    padding: var(--leo-spacing-l) var(--leo-spacing-2xl);
+  }
+
+  .empty-memories-table {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    align-self: stretch;
+    border-radius: var(--leo-radius-m);
+    border: 1px solid var(--leo-color-divider-subtle);
+  }
+
+  .empty-memories-empty-state {
+    display: flex;
+    width: 100%;
+    padding: var(--leo-spacing-4xl) 0px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .empty-memories-title {
+    color: var(--leo-color-text-tertiary);
+    text-align: center;
+    font-weight: 600;
+  }
+
+  .empty-memories-desc {
+    color: var(--leo-color-text-tertiary);
+    text-align: center;
+    font-weight: 400;
+  }
+
+  cr-dialog [slot="title"] {
+    color: var(--leo-color-text-primary);
+    font: var(--leo-font-heading-h3);
+    display: flex;
+    padding: var(--leo-spacing-2xl);
+    align-items: flex-start;
+    gap: var(--leo-spacing-xl);
+    align-self: stretch;
+  }
+
+  cr-dialog [slot="body"] {
+    display: flex;
+    padding: var(--leo-spacing-m) var(--leo-spacing-2xl);
+    flex-direction: column;
+    align-items: center;
+    gap: var(--leo-spacing-2xl);
+    align-self: stretch;
+  }
+
+  cr-dialog [slot="button-container"] {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-start;
+    gap: var(--leo-spacing-m);
+  }
+
+  /* Avoid stretching the buttons */
+  .action-box leo-button,
+  cr-dialog [slot="button-container"] leo-button {
+    width: auto !important;
+    min-width: unset !important;
+    flex: 0 0 auto !important;
+  }
+
+  leo-button.delete-all-button:not([isDisabled]) {
+    --leo-button-color: var(--leo-color-systemfeedback-error-text);
+    --primary-color: var(--leo-color-systemfeedback-error-background);
+    --bg: var(--leo-color-systemfeedback-error-background);
+  }
+
+  leo-button.delete-all-button[isDisabled] {
+    position: relative;
+    opacity: 0.65;
+    --leo-button-color: var(--leo-color-text-disabled);
+    border-radius: var(--leo-radius-full);
+  }
```

Comment:
nit: remove `isDisabled is automatic
```suggestion
```

Reply:
https://github.com/brave/brave-core/pull/30054/commits/8ed8849e997c58c40876c47ab2faa92f7c3ec3f2